### PR TITLE
Simon EBV (supersedes #49)

### PR DESCRIPTION
**Pull request by simonleandergrimm**
Supersedes #49 

Compared to #49 I:
 - Edited introduction
 - Reordered file to separate population variables from prevalence variables
 - Simplified calculation of US adult and non-adult population
 - Edited comments to `uk_seroprevalence_0_25` and added the estimate to estimate_prevalences
 - Edited name and CI of `nhanes_children_estimate`
 - Edited name and added CI for `nhanes_18_19_yo_estimate`
 - Refactored calculation of `denmark_seroprevalence`

### CODE REVIEW

File: pathogens/EBV.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+# also the question of mono being a proxy for EBV infection, or activation.
+
+pathogen_chars = PathogenChars(
+    na_type=NAType.DNA,
+    enveloped=Enveloped.ENVELOPED,
+    taxid=TaxID(10376),
+)
+
+
+under_18_population_US = Population(
```
[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/EBV.py:25]: Adopted!

[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/EBV.py]: I added methods to `Population` in `pathogen_properties.py` to make this work.

File: pathogens/EBV.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+    (4, 14): 0.625 * 100_000,
+    (15, 17): 0.8 * 100_000,
+    (18, 29): 0.86 * 100_000,
+    (30, 100): 0.95 * 100_000,
+}
+
+DENMARK_SEROPREVALENCE_SOURCE = "10.3109/inf.1983.15.issue-4.03"
+
+denmark_seroprevalences = {}
+for age_range in denmark_infection_rates:
```
[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/EBV.py]: Adopted.

File: pathogens/EBV.py
[the remark this answers is not in the document]
Comment (simonleandergrimm) [reply pathogens/EBV.py:1]: Adopted.

File: pathogens/ebv.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+us_fraction_u18 = Scalar(
+    scalar=0.222,
+    source="https://www.census.gov/quickfacts/fact/table/US/LFE046221#:~:text=%EE%A0%BF-,22.2%25,-Persons%2065%20years",
+)
+us_population_u18 = us_population(year=2022) * us_fraction_u18
+
+us_population_18plus = us_population(year=2022) - us_population_u18
+
+under_18_population_US = Population(
```
Comment (jeffkaufman) [inline pathogens/ebv.py:33]: delete

**Review by jeffkaufman — APPROVED** [review]
Almost ready!

**Pull request merged**